From db98cb29158741cc960f1e1a2df3d4214f5bd36e Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Tue, 6 Jun 2017 10:50:19 -0400
Subject: [PATCH] Ticket 49184 - adjust logging level in MO plugin

Description:  Change logging level for benign message

https://pagure.io/389-ds-base/issue/49184

Reviewed by: mreynolds(one line commit ruile)
---
 ldap/servers/plugins/memberof/memberof.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ldap/servers/plugins/memberof/memberof.c b/ldap/servers/plugins/memberof/memberof.c
index 5cd2c01..9bbe13c 100644
--- a/ldap/servers/plugins/memberof/memberof.c
+++ b/ldap/servers/plugins/memberof/memberof.c
@@ -3396,7 +3396,7 @@ int memberof_fix_memberof_callback(Slapi_Entry *e, void *callback_data)
 				/* This is quite unexpected, after a call to memberof_get_groups
 				 * ndn ancestors should be in the cache
 				 */
-				slapi_log_err(SLAPI_LOG_FATAL, MEMBEROF_PLUGIN_SUBSYSTEM, "memberof_fix_memberof_callback: Weird, %s is not in the cache\n", ndn);
+				slapi_log_err(SLAPI_LOG_PLUGIN, MEMBEROF_PLUGIN_SUBSYSTEM, "memberof_fix_memberof_callback: Weird, %s is not in the cache\n", ndn);
 			}
 		}
 	}
-- 
2.9.4

